core/src/main/java/jenkins/fingerprints/FingerprintStorage.java

```diff
@@ -102,14 +102,14 @@ public static FingerprintStorage getFileFingerprintStorage() {
      * For reference, see {@link FileFingerprintStorage#iterateAndCleanupFingerprints(TaskListener)}
      * For cleaning up the fingerprint {@link #cleanFingerprint(Fingerprint, TaskListener)} may be used.
      *
-     * @since TODO
+     * @since 2.246
```

Comment:
```suggestion
     * @since 2.248
```

Reply:
https://github.com/jenkinsci/jenkins/blob/jenkins-2.246/core/src/main/java/jenkins/fingerprints/FingerprintStorage.java#L105

2.246 and 2.247 weren't properly released, but this is the correct reference.